modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -460,7 +460,8 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::checkInputWindow(const GenericReal<is_ad>
                                                         const WindowFailure behavior,
                                                         const std::vector<Real> & global_limits)
 {
-  if (behavior != WindowFailure::WARN && behavior != WindowFailure::ERROR)
+  if (behavior != WindowFailure::WARN && behavior != WindowFailure::ERROR &&
```

Comment:
This is fine by me if @dschwen buys in. The only thing is this line should probably be `if if (behavior != WindowFailure::IGNORE && behavior != WindowFailure::EXTRAPOLATION)`,  and ...